### ISSUE TITLE
Any way to configure aliases?

### ISSUE THREAD

**Issue by azygis** [vuetifyjs/eslint-plugin-vuetify#87]
With `createVuetify` it's possible to specify aliases for virtual components. Is it possible to configure this plugin to use aliases as well?

**Comment by J-Sek** [comment]
Yes, you can rely on pnpm patch or [patch-package](https://www.npmjs.com/package/patch-package). I use it for some time and works like a charm.

1. open package code in second editor e.g. `code node_modules/eslint-plugin-vuetify`
2. edit whatever you want
3. run `ESLint: Restart ESLint Server` in VSCode or restart IDE if it does not have equivalent command
4. verify it applies new rules
5. `npx patch-package eslint-plugin-vuetify`

Hint for the props rules:
- auto-remove props with empty string
  - e.g. `persistentHint: '',`
- auto-rename prop
  - e.g. `label: 'text',`
- mark as deprecated
  - e.g. `textColor: false,`

This method does not count as "support" for custom rules, but is honestly very easy and does not require forking the repo.
